feasibility-triangle/troubleshooting/cql/5-Konsent.cql

```diff
@@ -0,0 +1,14 @@
+library "5-Konsent"
```

Comment:
should not be called Konsent - instead Consent. - or German Einwilligung

Reply:
I used the naming from Confluence. Perhaps we should change it there also?